src/connector.css

```diff
@@ -46,6 +46,11 @@
 	margin-top: 30px;
 }
 
+div#did-you-mean > p {
```

Comment:
@Basselbi Let's discuss how it should look like before adopting this; I am pretty confident we can change the approach (if needed) without requiring CSS.